Fix reference dir

tiagoengel opened the pull request:
Add back the logic to only verify screenshots and not record them when referenceDir is present. 

**NOTE**: I can't make the current project I'm working on work on version 9 (or current master) because of some internal error finding a resource. Would you be open to release a new version for 8 with this fix too? I can send a PR for that branch too, once this is merged.

fixes #166

xiphirx commented on .idea/codeStyleSettings.xml:
```diff
@@ -1,251 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Dont delete this please.

tiagoengel replied:
Sorry, didn't realise this. Done

xiphirx commented on plugin/src/main/kotlin/com/facebook/testing/screenshot/build/ScreenshotsPlugin.kt:
```diff
@@ -32,6 +32,8 @@ open class ScreenshotsPluginExtension {
   var multipleDevices = false
   /** The python executable to use */
   var pythonExecutable = "python"
+  /** The directory to pull screenshots from in verify only mode */
```
Lets change this to
```suggestion
  /** The directory to compare screenshots from in verify only mode */
```

tiagoengel replied:
Done

tiagoengel commented:
Sure, no big deal about the current master. I'll keep using my fork in the meantime and try to help when I have some free time. 

I've addressed all the comments 🙂
